.github/workflows/autocorrect-lint.yml

```diff
@@ -0,0 +1,36 @@
+name: AutoCorrect Lint
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "files/zh-cn/**"
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get changed files
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BASE_SHA: ${{ github.event.pull_request.base.sha }}
+          HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+        run: |
+          # Use the GitHub API to get the list of changed files
+          # documenation: https://docs.github.com/rest/commits/commits#compare-two-commits
+          DIFF_DOCUMENTS=$(gh api repos/{owner}/{repo}/compare/${{ env.BASE_SHA }}...${{ env.HEAD_SHA }} \
+            --paginate \
+            --jq '.files | .[] | select(.status|IN("added", "modified", "renamed", "copied", "changed")) | .filename')
+          # filter out files that are not markdown
+          DIFF_DOCUMENTS=$(echo "${DIFF_DOCUMENTS}" | egrep -i "^files/zh-cn/" | xargs)
+          echo "DIFF_DOCUMENTS=${DIFF_DOCUMENTS}" >> $GITHUB_ENV
+
+      - name: AutoCorrect changed content
+        if: ${{ env.DIFF_DOCUMENTS }}
+        uses: huacnlee/autocorrect-action@main
```

Comment:
can we specific the version here. Just ref `main` may bring some broken changes.

Reply:
There still have issue in by AutoCorrect lint result. The space background highlight can't shown on GitHub Action result.

I am working for fix that, after that. I will lock it on a version.

Reply:
Version just locked.

Reply:
> There still have issue in by AutoCorrect lint result. The space background highlight can't shown on GitHub Action result.
> 
> I am working for fix that, after that. I will lock it on a version.

This but is GitHub Action CSS bug. 

https://github.com/huacnlee/github-actions-color-test

Reply:
I'm not sure would a [problem matcher](https://github.com/orgs/community/discussions/25807) help with this. We have used it for [markdown lint](https://github.com/mdn/translated-content/blob/c070a9b09f1251ab6546514fd4b0e1b1ea026935/.github/workflows/markdownlint-problem-matcher.json).